monero-epee-bin-serde/src/lib.rs

```diff
@@ -0,0 +1,93 @@
+mod de;
+mod error;
+mod ser;
+mod varint;
+
+pub use crate::error::Error;
+
+use crate::de::Deserializer;
+use crate::ser::Serializer;
+use serde::de::DeserializeOwned;
+use serde::Serialize;
+use std::io::Read;
+
+pub type Result<T> = std::result::Result<T, Error>;
+
+const MAGIC: &[u8] = b"\x01\x11\x01\x01\x01\x01\x02\x01\x01";
```

Comment:
O.o What does this do? Maybe a comment would be good so you don't have to read `from_bytes`?